src/fetcher/serlo-api.tsx

```diff
@@ -2,9 +2,20 @@ import { request } from 'graphql-request'
 import { dataQuery, idQuery } from './query'
 import { processResponse } from './processResponse'
 
-const endpoint = 'https://api.serlo.org/graphql'
+const availableEndpoints = [
+  'https://api.serlo-development.dev/graphql',
+  'https://api.serlo-staging.dev/graphql',
+  'https://api.serlo.org/graphql'
+]
```

Comment:
```suggestion
const defaultEndpoint = 'https://api.serlo.org/graphql'
const availableEndpoints = [
  'https://api.serlo-development.dev/graphql',
  'https://api.serlo-staging.dev/graphql',
  defaultEndpoint
]
```

Reply:
I'd prefer if we'd use `api.serlo-development.dev` by default.